### PR TITLE
Update more Kubernetes Operator links from master to tagged versions

### PR DESCRIPTION
**Pull request by mdlinville**
Followup from #14620

Previews:
https://deploy-preview-14676--cockroachdb-docs.netlify.app/docs/v21.2/configure-cockroachdb-kubernetes.html?#ingress
https://deploy-preview-14676--cockroachdb-docs.netlify.app/docs/v22.1/configure-cockroachdb-kubernetes.html?#ingress
https://deploy-preview-14676--cockroachdb-docs.netlify.app/docs/v22.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.html?filters=eks#step-2-configure-your-network

### CODE REVIEW

File: v22.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md [hunk 1]
```diff
@@ -327,7 +327,7 @@ This important rule enables node communication between Kubernetes clusters in di
 
 The Kubernetes cluster in each region needs to have a [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html) pointed at its CoreDNS service, which you will configure in the next step.
 
-1. Upload our load balancer manifest [`dns-lb-eks.yaml`](https://github.com/cockroachdb/cockroach/blob/master/cloud/kubernetes/multiregion/eks/dns-lb-eks.yaml) to the Kubernetes clusters in all 3 regions:
+1. Upload our load balancer manifest [`dns-lb-eks.yaml`](https://github.com/cockroachdb/cockroach/master/cloud/kubernetes/multiregion/eks/dns-lb-eks.yaml) to the Kubernetes clusters in all 3 regions:
```
Comment (stbof) [inline v22.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md:330]: I get a 404 for this: https://github.com/cockroachdb/cockroach/master/cloud/kubernetes/multiregion/eks/dns-lb-eks.yaml

**Review by stbof — CHANGES_REQUESTED** [review]
First two are OK. Last is 404.

File: v22.1/configure-cockroachdb-kubernetes.md [hunk 2]
```diff
@@ -281,5 +281,7 @@ spec:
     Changing the SQL Ingress `host` on a running deployment will cause a rolling restart of the cluster, due to new node certificates being generated for the SQL host.
     {{site.data.alerts.end}}
 
-The [custom resource definition](https://github.com/cockroachdb/cockroach-operator/blob/master/config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml) details the fields supported by the Operator.
+{% capture latest_operator_version %}{% include_cached latest_operator_version.md %}{% endcapture %}
```
Comment (mdlinville) [inline v22.1/configure-cockroachdb-kubernetes.md:284]:
```suggestion
```

File: v22.1/configure-cockroachdb-kubernetes.md [hunk 3]
```diff
@@ -281,5 +281,6 @@ spec:
     Changing the SQL Ingress `host` on a running deployment will cause a rolling restart of the cluster, due to new node certificates being generated for the SQL host.
     {{site.data.alerts.end}}
 
-The [custom resource definition](https://github.com/cockroachdb/cockroach-operator/blob/master/config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml) details the fields supported by the Operator.
+
```
Comment (mdlinville) [inline v22.1/configure-cockroachdb-kubernetes.md:284]:
```suggestion
```

File: v22.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md (hunk 1 above)
Comment (mdlinville) [reply v22.1/orchestrate-cockroachdb-with-kubernetes-multi-cluster.md]: Done. I should have reverted that change (I had changed a bunch of them in error).

**Review by stbof — APPROVED** [review]

**Pull request merged**